commons/connector/health/health_activity.avsc

```diff
@@ -0,0 +1,24 @@
+{
+  "namespace": "org.radarcns.connector.health",
+  "name": "HKWorkoutTypeIdentifier_HKCategoryTypeIdentifierSleepAnalysis",
+  "type": "record",
+  "doc": "Activity , unit is activity type.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    {
+      "name": "timeReceived",
+      "type": "double",
+      "doc": "Time that the data was received from the Apple Healthkit ."
+    },
+    {
+      "name": "timeInterval",
+      "type": "int",
+      "doc": "Chronological window size (s)."
+    },
+    {
+      "name": "activity",
+      "type": "int",
+      "doc": "Activity taken in this period."
```

Comment:
Is there a defined set of possible activities that can be taken? If so, we can maybe add them as enums, similar to: https://github.com/RADAR-base/RADAR-Schemas/blob/master/commons/monitor/questionnaire/questionnaire_application_interaction_event.avsc#L12